### ISSUE TITLE
create `development` branch

### ISSUE THREAD

**Issue by susanssky** [leilafarsani/To-Do-List-Group-Work#15]
### What's the situation now?
>Issue: #10 & PR: #11 - Due to the lack of the 'type' parameter in the timestamp formatter function, the main branch cannot be deployed.
>Issue: #12 & PR: #13 - The Prisma Client needs to generate during the build process. Vercel has some quirks around dependency caching which prevent this. Therefore the build fails and the project is not deployed.

From the above, it can be seen that even if the local branch runs smoothly on localhost, once the local branch is merged into the `main` branch, it may not be successfully deployed on Vercel.

### Why does it need changing?
1. Until now, we only had one `main` branch to observe deployments/read logs. If there are any other issues with the `main` branch, we need to have an additional "copy".
2. We need a branch/website that can run/deploy successfully 100%. (`main` branch)

### What should it be like afterwards?
**Each person should be completely sure that the `development` branch is successfully deployed before merging the `development` branch into the `main` branch.**

>The `development` and `main` branch is now in the state of PR#13, so if there is a new PR(e.g. #16...) to be merged into the `development`branch first, and then merged into the `main` branch

the detailed process is as follows:
1. Write your code in VS Code.
2. Create a new branch in VS Code and push the new branch to the remote repo.
3. Open a PR to merge your new branch into the `development` branch, in order to check if your new branch can be deployed successfully on Vercel from the `development` branch.
4. If everything works properly after deployment, for example, if there are no strange behaviors of your new branch on the website, then open another PR to merge the `development` branch into the `main` branch.

**That's right, 2 PRs need to be created.**

# >>>Others<<<
**@leilafarsani needs to set the `development` branch as the **default branch** in the Github repository.**
**@leilafarsani needs to deploy the `development` branch on the Vercel website.**
- Since there is no need to "merge branches," there is no PR for this.

**Comment by bazmurphy** [comment]
Good job creating this ticket/issue.

In GitHub:

We also need to setup **branch protection** on `main` and possibly a minimum of `1 reviewer`
So that PRs cannot be automatically merged to `main` without a review.

We should also setup **branch protection** on `development` and possibly a minimum of `1 reviewer`
So that PRs cannot be automatically merged to `development` without a review.

I believe this can only be done by the Repository Owner.

**Comment by susanssky** [comment]
because i think everyone understand we have to push 'development' first so close
